src/main/resources/view/SuggestionsWindowView.fxml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.Insets?>
+
+<?import javafx.scene.control.Label?>
```

Comment:
Nit: Let's reorder and group these import statements together for clarity?

Reply:
Noted!